libreoffice-core/vcl/source/outdev/font.cxx

```diff
@@ -384,6 +387,49 @@ void OutputDevice::BeginFontSubstitution()
     pSVData->maGDIData.mbFontSubChanged = false;
 }
 
+void OutputDevice::AddCustomMacroFonts()
+{
+    OutputDevice* pDevice = Application::GetDefaultDevice();
+    pDevice->ImplAddCustomMacroFonts();
+}
+
+void OutputDevice::ImplAddCustomMacroFonts()
+{
+    std::list<std::string> fontNames = {
+        "Carlito",
+        "Carlito-Bold",
+        "Carlito-Italic",
+        "Carlito-BoldItalic",
+        "Caladea",
+        "Caladea-Bold",
+        "Caladea-Italic",
+        "Caladea-BoldItalic"
+    };
```

Comment:
OUStrings are utf-16 strings, to avoid prefer OUStringLiteral and `u"my_utf_16_constant"`

```suggestion
    static constexpr OUStringLiteral fontNames = {
        u"Carlito",
        u"Carlito-Bold",
        u"Carlito-Italic",
        u"Carlito-BoldItalic",
        u"Caladea",
        u"Caladea-Bold",
        u"Caladea-Italic",
        u"Caladea-BoldItalic"
    };
```

Reply:
This worked well, a bit verbose to get them into the map but works fine

Reply:
Ah, not quite what I had in mind. If the suggestion I gave didn't work verbatim that's on me, but you shouldn't have to use a list.

Reply:
Hmm, using fontNames[] gives error `cannot form array of deduced class template specialization type` - something funky about OUStringLiteral here. I don't think it's a huge deal to list it if it works here, but I also just like to dodge C++ template weirdness.

Reply:
I know this sounds weird, but it slows down startup time to initialize these types, so easy wins here are better.

Seems like they messed up the literal type a bit, in that case use `std::u16stringview` instead.

`static constexpr std::u16stringview fontNames[] { u"a", u"b", u"c" };`